frontend/src/components/samples/SamplesAssociatedProjects.tsx

```diff
@@ -38,13 +39,15 @@ const SamplesAssociatedProjects = ({
   useEffect(() => {
     if (sampleID) {
       const filterKey = 'project_derived_by_samples__sample__id'
-		  const filter: FilterSetting = createFixedFilter(FILTER_TYPE.INPUT_OBJECT_ID, filterKey, sampleID)
-      setSampleIDFilter(filter)
+		  const filter: FilterSetting = createFixedFilter(FILTER_TYPE.INPUT_OBJECT_ID, filterKey, sampleID.toString())
+      dispatch(ProjectsOfSamplesActions.setFixedFilter({...filter}))
+      setSampleIDFilter({...filter})
+      dispatch(ProjectsOfSamplesActions.setStale(true))
     }
   }, [sampleID])
 
```

Comment:
My linter is not pleased with the lack of dispatch in the deps.

Reply:
ok i'll make him happy ... should not hurt anything ;)